db/client.go

```diff
@@ -0,0 +1,42 @@
+package db
+
+import (
+	"Car-parking-management-systems/models"
+	"log"
+)
+
+// DataStore is an interface for query ops
+type DataStore interface {
+	AddCar(car *models.Car) (string, error)
+	GetCar(id string) (*models.Car, error)
+	DeleteCar(id string) error
+	UpdateCar(car *models.Car) error
```

Comment:
no need to have a separate method for updation we can use SaveCar for updation as well as creation